src/index.ts

```diff
@@ -2,3 +2,4 @@
 
 export * from './AssetsTransferApi';
 export * from './constructApiPromise';
+export { Format, ITransferArgsOpts, TxResult } from './types';
```

Comment:
We need to get rid of these export types, I know that it improves the layout of the docs, but to keep a consistent SDK we need to ensure for now, we dont export them

Reply:
Sounds good, I'll remove those and push up the changes.